frontend/src/components/Forms/CoffeeChatsForm/CoffeeChatsForm.tsx

```diff
@@ -0,0 +1,176 @@
+      Emitters.generalError.emit({
+        headerMsg: 'Cannot Coffee Chat Yourself',
+        contentMsg: 'Please submit a coffee chat with another member!'
+      });
+    } else if (await coffeeChatExists()) {
+      Emitters.generalError.emit({
+        headerMsg: 'Coffee Chat Exists',
+        contentMsg: 'Please submit a coffee chat with a new member!'
+      });
+    } else {
+      const newCoffeeChat: CoffeeChat = {
+        uuid: '',
+        submitter: userInfo,
+        otherMember: member,
+        slackLink,
+        category,
+        status: 'pending' as Status,
+        date: new Date().getTime()
+      };
+
+      CoffeeChatAPI.createCoffeeChat(newCoffeeChat).then(() => {
+        setPendingChats((pending) => [...pending, newCoffeeChat]);
+      });
+      Emitters.generalSuccess.emit({
+        headerMsg: 'Coffee Chat submitted!',
+        contentMsg: `The leads were notified of your submission, and your coffee chat will be approved soon!`
+      });
+      setMember(undefined);
+      setCategory('');
+      setSlackLink('');
+    }
+  };
+
+  return (
+    <div>
+      <Form className={styles.form_style}>
+        <h1>Submit Coffee Chats</h1>
+        <p>
+          Submit your coffee chats to earn credit for the Coffee Chat Bingo! Fill out this form
+          after each chat to receive your credit.
+        </p>
+        <div className={styles.inline}>
+          <label className={styles.bold}>
+            Select a team member: <span className={styles.red_color}>*</span>
+          </label>
+          <div className={styles.center_and_flex}>
+            {membersList ? (
+              <Dropdown
+                placeholder="Select a team member"
+                fluid
+                search={(options, query) =>
+                  options.filter((option) => option.key.toLowerCase().includes(query.toLowerCase()))
+                }
+                selection
+                value={member?.netid ?? ''}
+                text={member ? `${member.firstName} ${member.lastName}` : ''}
+                options={membersList
+                  .sort((m1, m2) =>
+                    `${m1.firstName} ${m1.lastName}`.localeCompare(`${m2.firstName} ${m2.lastName}`)
+                  )
+                  .map((member) => ({
+                    key: member.netid,
+                    label: (
+                      <div className={styles.flex_start}>
+                        {member.firstName} {member.lastName}
+                      </div>
+                    ),
+                    value: member.netid
+                  }))}
+                onChange={(_, data) => {
+                  setMember(membersList.find((member) => member.netid === data.value));
+                }}
+              />
+            ) : undefined}
+          </div>
+        </div>
+        <div className={styles.inline}>
+          <label className={styles.bold}>
+            Which category does this coffee chat fulfill?{' '}
```

Comment:
What if we made this a dropdown? Let's hardcode the categories somewhere, probably here:
https://github.com/cornell-dti/idol/blob/5bb5d22ce0c0d1680bb39a0e83e9bb86c24ae697/frontend/src/consts.ts#L1-L6
and then we can pull that data in here. This will help us set up some of the verification logic.